src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -368,8 +368,8 @@ private function getOrderProducts(Order $order): OrderProductsForViewing
             $product['quantity_refundable'] = $product['product_quantity'] - $product['product_quantity_return'] - $product['product_quantity_refunded'];
             $product['amount_refundable'] = $product['total_price_tax_excl'] - $product['total_refunded_tax_excl'];
             $product['amount_refundable_tax_incl'] = $product['total_price_tax_incl'] - $product['total_refunded_tax_incl'];
-            $product['displayed_max_refundable'] = $taxCalculationMethod === PS_TAX_EXC ? $product['amount_refundable'] : $product['amount_refundable_tax_incl'];
-            $resumeAmountKey = $taxCalculationMethod === PS_TAX_EXC ? 'total_refunded_tax_excl' : 'total_refunded_tax_incl';
+            $product['displayed_max_refundable'] = $order->getTaxCalculationMethod() ? $product['amount_refundable'] : $product['amount_refundable_tax_incl'];
```

Comment:
Not sure if this change is right
@jolelievre can you tell me ?

Reply:
Nope this one I fixed there was a bug, the correct code is the one from 177x